importers/scripts/importStepLibrary.ts

```diff
@@ -0,0 +1,89 @@
+import { createWriteStream, mkdirSync } from 'fs'
+import { get } from 'http'
+import { S3 } from '@aws-sdk/client-s3'
+import { BeDatabaseCreator, BeImportFileCreator } from '../src';
+import { SwordImporter } from './../src/bible/sword/src/importer';
+import { ConnectionOptions } from 'typeorm';
+
+const BUCKETS = [
+    'tyndale-house-public',
+    'tyndale-house-private'
+]
+const REGION = 'eu-west-1'
+const LOCAL_CACHE_PATH = 'data/step-library'
+
+const CONNECTION_OPTIONS: ConnectionOptions = {
+    type: 'better-sqlite3',
+    database: 'bibles.db'
```

Comment:
is the idea to put all available sword modules in the sqlite database and preload that in the app package? i think the resulting sqlite file will be too large. for the server this will make sense though.
if we want to preload many versions i suggest creating BEF files and preload those. BEF just bundles what you can otherwise also fetch from the BE server API. That means with a single logic you can then import new versions either from pre-packaged BEF or from the API. I have the BEF import running in the Emmaus app and it works invisibly for the user. i can walk you through it if you want (there is also some discussing about this somewhere on Slack).
(for reference: my BEF files are between 2 and 3 MB, that is including notes / intros / sections / crossrefs. i strip strongs though, since i don't need them currently, so i would expect ESV+Strongs to be more like 4 to 5MB.)

Reply:
Yes, I can create BEF files, that is probably the best way

I only set it up as one large database for testing purposes